packages/dotnet.rb

```diff
@@ -0,0 +1,44 @@
+require 'package'
+
+class Dotnet < Package
+  description ''
```

Comment:
Missing description?

Reply:
Added. Forgot to add this change to the commit.